development/makefile

```diff
@@ -44,3 +44,9 @@ portstype.o:  portstype.cpp
 translator.o: translator.cpp
 	gcc -c translator.cpp
 
+.PHONY: clean
```

Comment:
Note that .PHONY is a gnu-make special built-in target (see [Phony Targets](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html)).  May need to ensure that this doesn't create any unintended consequences for use with other C++ build environments.